actions/settings.py

```diff
@@ -141,4 +144,30 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
+
+# CSP
+# https://django-csp.readthedocs.io/en/latest/configuration.html
+CSP_REPORT_ONLY = DEBUG
+CSP_DEFAULT_SRC = ["'self'"]
```

Comment:
This can also pull in images from user-images.githubusercontent.com

e.g https://actions.opensafely.org/actions/cohort-report/v3.0.0/

Reply:
Good spot, that's been added ✅